### DISCUSSION TITLE
(WHAT IF) Computer generator/builder out of 86box/PCem?

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#1454] [General]
What if someone were to operate a service where you would submit a configuration file (.cfg) made in 86box or PCem, and they would build a computer or PC based on your configuration file? For instance, imagine taking a configuration of an 86Box VM and turning it into a real-life computer?

**Comment by bachimusprime** [comment]
That would be far too expensive for us to create.

**Reply by jadasse** [reply]
I didn't mean you guys... (no offense)

**Reply by bachimusprime** [reply]
Then why did you post this on our discussion board? Sorry, it's just that it makes it seem like you're asking US to do that... No offense...

**Reply by jadasse** [reply]
I thought that no one would recognize it in the crazy ideas subreddit... i could be wrong

**Reply by jadasse** [reply]
And i was right

**Reply by OBattler** [reply]
Such an idea would be highly impracticable, as it would require a chip factory with everything needed to manufacture all those old boards and cards, which would also make the end result very expensive.

**Reply by jadasse** [reply]
> Such an idea would be highly impracticable, as it would require a chip factory with everything needed to manufacture all those old boards and cards, which would also make the end result very expensive.

Makes sense.
